app/assets/stylesheets/earthworks.css

```diff
@@ -205,3 +205,9 @@ Uncomment lines 142-144
   color: var(--stanford-digital-blue-dark) !important;
   border-color: var(--stanford-digital-blue-dark) !important;
 }
+
+:focus-visible {
+  outline: 1px solid black!important; /* Outer color */
+  box-shadow: 0 0 0 2px white!important; /* Inner color */
```

Comment:
Had to include !important because of these bootstrap lines in .nav-link https://github.com/twbs/bootstrap/blob/048f56f51460df75e92a2f7b472e1c56baeb68f7/scss/_nav.scss#L41-L44